docs/kratos/emails-sms/05_custom-email-templates.mdx

```diff
@@ -0,0 +1,347 @@
+
+:::
+
+Use these variables in templates:
+
+- `recovery.valid` - `To`,`RecoveryURL`, and `Identity` for validating recovery.
+- `recovery.invalid` - `To` for invalidating recovery.
+- `verification.valid` - `To`, `VerificationURL` and `Identity` for validating verification.
+- `verification.invalid` - `To` for invalidating verification.
+
+Each template must have two versions: `html` and `plaintext`.
+
+- `html` version uses the HTML syntax to achieve the desired look and functionality (such as clickable buttons) of the email
+  message.
+- `plaintext` version can't contain any HTML. Must contain only plain text content and any necessary gotmpl logic. This version is
+  used as a fallback when the `html` version cannot be delivered, for example when the user's mail server blocks HTML in all
+  incoming messages.
+
+For example, for this template:
+
+[`https://github.com/ory/kratos/blob/master/courier/template/courier/builtin/templates/verification/valid/email.body.gotmpl`](https://github.com/ory/kratos/blob/master/courier/template/courier/builtin/templates/verification/valid/email.body.gotmpl)
+
+- The `html` version looks like this:
+
+  ```gotmpl title="courier/template/templates/verification/valid/email.body.gotmpl"
+  Hi, please verify your account by clicking the following link:
+
+  <a href="{{ .VerificationURL }}">{{ .VerificationURL }}</a>
+  ```
+
+- The `plaintext` version doesn't use any HTML and keeps only the gotmpl logic (`{{ .VerificationURL }}`) and plain text.
+
+  ```gotmp title="courier/template/templates/verification/valid/email.body.plaintext.gotmpl"
+  Hi, please verify your account by clicking the following link: {{ .VerificationURL }}
+  ```
+
+## Customizing template content for specific users
+
+To enable customizing the content of templates based on the identity of the recipient of the email, the Ory identity is available
+as the `Identity` object. This object is a map containing all the attributes of an identity defined in the identity schema, such
+as `id`, `state`, `recovery_addresses`, `verifiable_addresses` and `traits`.
+
+:::tip
+
+Read [this document](../manage-identities/overview) to learn more about the Ory identity and the identity schema.
+
+:::
+
+### Nested templates: i18n customization
+
+You can use nested templates to render `email.subject.gotmpl`, `email.body.gotmpl` and `email.body.plaintext.gotmpl` templates.
+
+To enable i18n customization of templates, customize the identity schema to include the user's preferred communication language.
+For example:
+
+```json title="Sample custom Identity Schema with user language"
+{
+  // ...
+  "properties": {
+    "traits": {
+      "type": "object",
+      "properties": {
+        // ...
+        // highlight-start
+        "lang": {
+          "type": "string",
+          "title": "Your preferred language"
+      },
+      // highlight-end
+      "required": [
+        "email"
+      ],
+      "additionalProperties": false,
+    }
+  }
+}
+```
```

Comment:
These can also be added as "meta" attributes so then you don't need to rely on the user supplying the language attribute, but could for instance do this based on location or system language.
https://www.ory.sh/docs/guides/manage-identities/managing-users-identities-metadata

Reply:
just maybe link to meta and say this can be done, no need to provide more examples i think